### PR TITLE
Content: Cats Vs Rattlesnake Plains Hunting Patrol

### PR DESCRIPTION
**Pull request by Artyphex**
Plains exclusive, greenleaf exclusive. Your patrol encounters a rattlesnake, if you have high enough stats you kill it, if not, things go poorly.

### CODE REVIEW

**Review by Tiritirimatangi — COMMENTED** [review]
You need to delete the groupmice patrol (which has the old patrol formatting and won't work), and make sure the patrol passes all the checks - currently you're failing the encoding test, which means there's non-ascii characters in your new content?

**Comment by Artyphex** [comment]
> You need to delete the groupmice patrol (which has the old patrol formatting and won't work), and make sure the patrol passes all the checks - currently you're failing the encoding test, which means there's non-ascii characters in your new content?

Yup! I'm fixing it now. It's the apostrophes

File: resources/dicts/patrols/plains/hunting/greenleaf.json
Comment (Tiritirimatangi) [inline resources/dicts/patrols/plains/hunting/greenleaf.json:1]: You also need to specify that the leader can't be on the patrol by saying "leader": [0, 0],

**Review by Tiritirimatangi — APPROVED** [review]
Looks like you're good to go! Excellent job!

**Comment by Artyphex** [comment]
> Looks like you're good to go! Excellent job!

Woo! Thank you!

**Pull request merged**